### PR TITLE
Upload page redesigned and re implemented 

### PR DESCRIPTION
**Pull request by tewarig**
This pr solves #33 
as discuss this pr adds

 sidebar for navigation (animated)
 separate upload button
 own image gallery (responsive)
 click to zoom feature (single click to zoom in and zoom out)
 newly updated image gallery and upload functionality.
also, this fixes issue which was found in  #39 
Images

### CODE REVIEW

**Comment by pkesavap** [comment]
Once the user has logged in, it should reroute to the upload page where the user can upload a photo.
Right now, its being reloaded to /

@tewarig

**Review by pkesavap — CHANGES_REQUESTED** [review]
Add reroute for upload after user logs in

File: app.py [hunk 1]
```diff
@@ -117,7 +118,7 @@ def login():
                 else:
                     return "error"
                 flash("You are now login", "success")
-                return redirect(url_for('home'))
+                return redirect(url_for('upload'))
             else:
                 flash("Incorrect password", "danger")
                 return render_template("login.html")
```
Comment (tewarig) [inline app.py:124]: hi, it is already there if user login, he would be redirected to the upload route.

**Review by tewarig — COMMENTED** [review]
hi,  @pkesavap  check this he is being redirected to the upload route

**Comment by tewarig** [comment]
<iframe width="560" height="315" src="https://www.youtube.com/embed/Ly8aE130b14" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen={true}></iframe> 
here is a preview of that

**Comment by pkesavap** [comment]
Ah my bad cookie problem. ;)

LGTM

**Pull request merged**